src/app/TempConverter/TempConverter.c

```diff
@@ -0,0 +1,45 @@
+#include "TempConverter.h"

```

Comment:
See the github jenkins check- it failed. Probably because sconstruct.py looks for a test_TempConverter.c file which doesn't exist yet